src/vcpkg/vcpkgpaths.cpp

```diff
@@ -484,7 +496,7 @@ namespace vcpkg
                                                                                             : RequireExactVersions::NO))
                 , m_env_cache(m_ff_settings.compiler_tracking)
                 , triplets_dirs(
-                      Util::fmap(args.overlay_triplets,
+                      Util::fmap(args.cli_overlay_triplets,
```

Comment:
Are these the ones we need here? I didn't know which to put 🆘

Reply:
hard to say without looking at callsites for `triplets_dir`, but I guess we should set it to the merged overlay triplets.